frontend/components/notifications/SubscriptionDialog.vue

```diff
@@ -0,0 +1,232 @@
+    lastSaveFailed = true // we will try again when the dialog closes
+  } else {
+    lastSaveFailed = false
+  }
+}
+
+const isOptionAvailable = (key: string) => props.value?.premiumUser || !(key in DefaultValueOfValidatorOptionsNeedingPremium || key in DefaultValueOfAccountOptionsNeedingPremium)
+
+function closeDialog () : void {
+  if (lastSaveFailed) {
+    // second chance: we try not to lose what the user has set
+    sendUserPreferencesToAPI()
+  }
+  dialogRef?.value.close(true)
+}
+</script>
+
+<template>
+  <div v-if="props && tPath" class="content">
+    <div class="title">
+      {{ t('notifications.subscriptions.dialog_title') }}
+    </div>
+
+    <div v-if="t(tPath+'explanation')" class="explanation">
+      {{ t(tPath+'explanation') }}
+    </div>
+
+    <div v-if="props.validatorSub">
+      <NotificationsSubscriptionRow v-model="modifiableOptions.offlineValidator" :t-path="tPath+'offline_validator'" :lacks-premium-subscription="!isOptionAvailable('offlineValidator')" class="row" />
+      <NotificationsSubscriptionRow
+        v-model="modifiableOptions.offlineGroup"
+        :t-path="tPath+'offline_group'"
+        :lacks-premium-subscription="!isOptionAvailable('offlineGroup')"
+        input-type="percent"
+        :default="DefaultValueOfValidatorOptionsNeedingPremium.offlineGroup"
+        class="row"
+      />
+      <NotificationsSubscriptionRow
+        v-model="modifiableOptions.missedAttestations"
+        :t-path="tPath+'missed_attestations'"
+        :lacks-premium-subscription="!isOptionAvailable('missedAttestations')"
+        class="row"
+        :value-in-text="Math.round(networkInfo.secondsPerSlot*networkInfo.slotsPerEpoch/6)/10"
+      />
+      <NotificationsSubscriptionRow v-model="modifiableOptions.proposedBlock" :t-path="tPath+'proposed_block'" :lacks-premium-subscription="!isOptionAvailable('proposedBlock')" class="row" />
+      <NotificationsSubscriptionRow v-model="modifiableOptions.upcomingProposal" :t-path="tPath+'upcoming_proposal'" :lacks-premium-subscription="!isOptionAvailable('upcomingProposal')" class="row" />
+      <NotificationsSubscriptionRow v-model="modifiableOptions.syncCommittee" :t-path="tPath+'sync_committee'" :lacks-premium-subscription="!isOptionAvailable('syncCommittee')" class="row" />
+      <NotificationsSubscriptionRow v-model="modifiableOptions.withdrawn" :t-path="tPath+'withdrawn'" :lacks-premium-subscription="!isOptionAvailable('withdrawn')" class="row" />
+      <NotificationsSubscriptionRow v-model="modifiableOptions.slashed" :t-path="tPath+'slashed'" :lacks-premium-subscription="!isOptionAvailable('slashed')" class="row" />
+      <NotificationsSubscriptionRow v-model="modifiableOptions.realTime" :t-path="tPath+'real_time'" :lacks-premium-subscription="!isOptionAvailable('realTime')" class="row" />
+      <hr>
```

Comment:
`<hr>` has the color given by Figma I believe.
But [it seems that this tag is more or less deprecated](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/hr) so I replaced it with a `<div/>`

Reply:
better, but couldn't you also set the border top of the 'top' row instead of adding an extra div? (if too complicated please ignore)

Reply:
As you can see here
![image](https://github.com/user-attachments/assets/92a2ac47-d3ee-4256-890b-ce1ed04fc21e)
the bottom of the rows is not in the middle between two rows.

It might be possible to change the whole CSS to do what you want but should I spend time trying to change something that works and is simple?